### DISCUSSION TITLE
Time steps in Smokeview

### DISCUSSION THREAD

**Discussion by DOGzym** [firemodels/fds#12921] [Q&A] [answered]
Hello all,

The default time step is used in the jet fire model and solid fire spread model I built respectively, but the time step shows a big difference when pulling the progress bar in Smokeview: 0.03s and 1.5s respectively.

What I want to know is, what factors determine the time step in Smokeview? Can the time step in Smokeview be customized in code?

**Comment by TristanHehnen** [comment, marked as answer]
Output frequencies can be controlled via the `DUMP` namelist. The default is `NFRAMES=1000`, which divides the simulation time into 1000 data points. Thus, if the runtime of the simulations is different, the time step size between the data points is different. 

The frequencies are controlled globally with `NFRAMES`, but can also be adjusted individually for example for `SLCF` using `DT_SLCF`.

Have a look in the [users guide](https://github.com/firemodels/fds/releases/tag/FDS-6.9.1), section "22.1 Controlling the Frequency of Output".

**Reply by DOGzym** [reply]
Thanks a lot!

**Comment by gforney** [comment]
The times that smokeview didplays should be the same as what fds outputs to
it. Podt your input file and ill take a look when i get back next week
